internal/component/prometheus/relabel/relabel.go

```diff
@@ -259,6 +272,9 @@ func (c *Component) relabel(val float64, lbls labels.Labels) labels.Labels {
 	// Set the cache size to the cache.len
 	// TODO(@mattdurham): Instead of setting this each time could collect on demand for better performance.
 	c.cacheSize.Set(float64(c.cache.Len()))
+
+	c.debuggingStreamHandler.Stream(c.opts.ID, fmt.Sprintf("%s => %s", lbls.String(), relabelled.String()))
```

Comment:
Why is this function called `Stream`? 

To stream a piece of information seems to be a name suitable for sending a sequence of data in a way that parts of it are being sent at a time, often asynchronously. 

Here we are not sending a sequence of data and there is one, synchronous call. Maybe call it `Publish` or `Push` as it is adding to a potentially active stream.